datafusion/core/tests/fifo.rs

```diff
@@ -394,8 +491,6 @@ mod unix_test {
                 a2  INT NOT NULL
             )
             STORED AS CSV
-            WITH HEADER ROW
```

Comment:
There is no particular reason that CSV headers could not be supported, but it was simpler to just not read/write them for the purposes of this test